README.md

```diff
@@ -27,7 +27,8 @@ require('laravel-elixir-bless');
 
 elixir(function(mix) {
     mix.bless('app.css', {
-        imports: false
+        imports: false,
+        watch: './resources/less/*.less'
```

Comment:
That is only an example of a pattern that can be provided via the option. Elixir's `watch()` is internally using [gulp watch](https://www.npmjs.com/package/gulp-watch) so any pattern accepted by that can be provided (e.g. a directory, a file, file patterns, etc).